deployment/constructs/lambda-dynamo-event.ts

```diff
@@ -28,50 +29,14 @@ export class LambdaDynamoEventSource extends Construct {
     const {
       additionalPolicies = [],
       memorySize = 1024,
-      tables,
+      eventSourceTables,
+      grantTables = [],
```

Comment:
Feels like these will always be present and always be the same

Reply:
you're right.